src/hotspot/share/nmt/vmatree.hpp

```diff
@@ -233,6 +251,20 @@ class VMATree {
   void print_on(outputStream* out);
 #endif
 
+  template<typename F>
+  void visit_range_in_order(const position& from, const position& to, F f) {
+    _tree.visit_range_in_order(from, to, f);
+  }
+
+  VMATreap& tree() { return _tree; }
+
+  void print_self() {
+    visit_in_order([&](TreapNode* current) {
+      tty->print("(%s) - %s - ", NMTUtil::tag_to_name(current->val().out.mem_tag()), statetype_to_string(current->val().out.type()));
+      return true;
+    });
+    tty->cr();
+  }
```

Comment:
This can be removed, I'm rather sure(?)

Reply:
Only print_self is removed. The other two methods are used in the code.